src/DataMigration/custom/Cmdlets/Get-AzDataMigrationAssessment.ps1

```diff
@@ -0,0 +1,93 @@
+function Get-AzDataMigrationAssessment 
+{
+    [OutputType()]
+    [CmdletBinding(PositionalBinding=$false)]
+    [Microsoft.Azure.PowerShell.Cmdlets.DataMigration.Description('Runs assessment on given Sql Servers')]
+
+    param(
+        [Parameter(ParameterSetName='CommandLine', Mandatory, HelpMessage='Sql Server Connection Strings')]
+        [System.String[]]
+        ${ConnectionStrings},
+
+
+        [Parameter(ParameterSetName='CommandLine', HelpMessage='Output folder to store assessment report')]
+        [System.String]
+        ${OutputFolder},
+
+
+        [Parameter(ParameterSetName='CommandLine', HelpMessage='Enable this parameter to overwrite the existing assessment report')]
+        [System.Management.Automation.SwitchParameter]
+        ${Overwrite},
+
+        [Parameter(ParameterSetName='ConfigFile', Mandatory, HelpMessage='Path of the ConfigFile')]
+        [System.String]
+        ${ConfigFilePath}
+    )
+
+    process 
+    {
+        try 
+        {
+            #Defining Base and Exe paths
+            $BaseFolder = $PSScriptRoot;
+            $ExePath = Join-Path -Path $BaseFolder -ChildPath SqlAssessment.Console.csproj\SqlAssessment.exe;
```

Comment:
When the module is used by customer, the path is not valid

Reply:
What will happen to linux and mac users?

Reply:
Thanks for informing this. We will move it to user profile for all OS platforms. For Windows LocalAppData and Mac/OSX  and Linux we will have it in UserProfile.

Reply:
Please let me know if you have any other suggestion on this.